compiler/rustc_codegen_llvm/src/intrinsic.rs

```diff
@@ -195,6 +197,11 @@ impl<'ll, 'tcx> IntrinsicCallBuilderMethods<'tcx> for Builder<'_, 'll, 'tcx> {
                 codegen_autodiff(self, tcx, instance, args, result);
                 return Ok(());
             }
+            sym::offload => {
+                // FIXME(Sa4dUs): emit error when offload is not enabled
```

Comment:
I'd rather have that as a TODO, thus fixed before we land it. You can just copy the check I had above (and potentially error if you find an intrinsic, but not the flag set). The current main already requires it, and we shouldn't change nightly behaviour because of an experimental feature without a user setting a feature flag.